tools/vmlist.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python
+
+import ConfigParser
+import docopt
+import multiprocessing
+import novaclient.client
+import os
+import subprocess
+import sys
+import tempfile
+import textwrap
+
+CACHEFILE = "~/.vmlist.cache"
+CONFFILE = "~/.vmlist.conf"
+
+# mira004.front.sepia.ceph.com is dead
+
+VMMACHINES = textwrap.dedent('''\
```

Comment:
Name change is fine with me; env vars: well, you can set it in the .conf file too;  I guess we could define an order env -> conf -> default.  You think others of the things should also be in the env, or just the hosts?  (note also the -m switch)